build.gradle

```diff
@@ -365,6 +372,7 @@ subprojects {
                  'opencensus-contrib-http-util',
                  'opencensus-contrib-log-correlation-stackdriver',
                  'opencensus-contrib-monitored-resource-util',
+                 'opencensus-contrib-spring',
```

Comment:
Could you please add an entry to the changelog?

Reply:
done